internetarchive/iaux-book-visual-adjustments
# Updates: adds optional header block, bolsters event details

This PR includes the following:

- Updates custom event context
  - has more details to send `{ activeCount, options, changedOptionId }`
- Unit Test
  - new events `describe` block
  - add and update tests around event firing
- Update to demo so that we show the following:
  - starting with a visual adjustment on
  - shows with or without header

To Test:
- pull down repo
- run commands `npm install && npm start`
- header is toggled through to see native component capability
- on click, header updates active options
- open dev tools: custom event emitted shows the updated data

![ezgif com-video-to-gif (6)](https://user-images.githubusercontent.com/7840857/93397598-c6759480-f82e-11ea-9e67-e79b455fcf89.gif)
